src/models/Resource.ts

```
@@ -0,0 +1,30 @@
+      type: String,
+      enum: ['available', 'booked', 'needs_maintenance', 'offline'],
+      default: 'available',
+    },
+    maintenanceInterval: { type: Number, required: true, default: 50 },
+    usageCount: { type: Number, default: 0 },
+    usageDurationHours: { type: Number, default: 0 },
+  },
+  { timestamps: true }
+);
+
+export const Resource = model<IResource>('Resource', ResourceSchema);
```

ESLint

[error] Replace `'Resource'` with `"Resource"`

(prettier/prettier)